index.bs

```diff
@@ -5988,21 +5988,74 @@ The following extended attributes are <dfn for="extended attributes">applicable
         production, [=set/append=] each of the [=extended attributes=] present in the production's
         <emu-nt><a href="#prod-ExtendedAttributeList">ExtendedAttributeList</a></emu-nt> to
         |extended attributes|.
+
+        <div class="example">
+            <pre class="idl">
+                interface I {
+                    attribute [XAttr] long attrib;
+                    void f1(sequence<[XAttr] long> arg);
+                    void f2(optional [XAttr] long arg);
+
+                    maplike&lt;[XAttr2] DOMString, [XAttr3] long>;
+                };
+
+                dictionary D {
+                    required [XAttr] long member;
+                };
+            </pre>
+        </div>
     1.  If |type| is a [=member type=] of a [=union type=] |U|, [=set/append=] each of the
         [=extended attributes associated with=] |U| to |extended attributes|.
+
+        <div class="example">
+            <pre class="idl">
+                interface I {
+                    attribute [XAttr] (long or Node) attrib;
+                };
+            </pre>
+        </div>
     1.  If |type| appears as part of a <emu-nt><a href="#prod-Type">Type</a></emu-nt> production
-        within an <emu-nt><a href="#prod-Argument">Argument</a></emu-nt> production, [=set/append=]
-        to |extended attributes| all of the [=extended attributes=] present in the production's
+        directly within an <emu-nt><a href="#prod-Argument">Argument</a></emu-nt> production,
+        [=set/append=] to |extended attributes| all of the [=extended attributes=] present in the
+        production's
         <emu-nt><a href="#prod-ExtendedAttributeList">ExtendedAttributeList</a></emu-nt> that are
         [=applicable to types=].
+
+        <div class="example">
+            <pre class="idl">
+                interface I {
+                    void f([XAttr] long attrib);
+                };
+            </pre>
+
+            but only if [<code>XAttr</code>] is [=applicable to types=]; otherwise
```

Comment:
The lowercase "but" here is a bit weird. Maybe uppercase it or give a tad more context.